great-idea-website/index.html

```diff
@@ -13,51 +13,88 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
+<header>
+    <nav>
```

Comment:
nice work you code is clean and has a proper format. I would suggest isolating each piece of the page as its own piece. For example I would wrap the nav and img in a div with a class name of "static-navbar" and then also give the img its own div